pages/common/virt-install.md

```diff
@@ -3,6 +3,26 @@
 > Create virtual machines with libvirt and begin OS installation.
 > More information: <https://virt-manager.org/>.
 
-- Create a virtual machine with 1 GiB RAM and 12 GiB storage and start Debian installation:
+- Create a virtual machine with 1 GiB RAM and 12 GiB storage and start a Debian installation:
```

Comment:
Not related to this PR but we may fix the unit symbol (see https://en.wikipedia.org/wiki/Gigabyte):
```suggestion
- Create a virtual machine with 1 GB RAM and 12 GB storage and start a Debian installation:
```

Reply:
(same comment for units below)

Reply:
Thanks for the correction